### PR TITLE
Help Center: add forum option

### PR DESCRIPTION
**Pull request by arcangelini**
#### Proposed Changes

This adds the Forums as an option to the help center contact page. Previously we only forwarded free users to this option. This adds a new button that shows/hides all the support options with the exception of the Forums at the top.

| Chat Closed | Chat Open |
| - | - |
| Markup 2023-02-09 at 13 13 27 | Markup 2023-02-09 at 13 14 33 |

#### Testing Instructions

1. Pull branch and start
2. Open up Help Center and like "Still need help"
3. You should see the forum option regardless of your plan
4. Free users should be forwarded to the forum op
5. Test in atomic using the plugin

[My test topic that was submitted](https://wordpress.com/forums/?post_type=topic&p=3919173&view=all)

### CODE REVIEW

**Review by alshakero — COMMENTED** [review]
Besides the color. This looks good 👍🏼 

We can't ship it yet though, paid sites will probably have issues on the contact form in forums mode. We have to fix the form first.

**Comment by arcangelini** [comment]
Oh just realized we will need to proxy the forum topic creation endpoint to work in Atomic

**Comment by kriskarkoski** [comment]
> We can't ship it yet though, paid sites will probably have issues on the contact form in forums mode. We have to fix the form first.

@alshakero Can you clarify what the issues are? We're sending add-on only users to the forums as of earlier this month and I'm not aware of issues. Or did you mean Atomic specifically, which Tony mentioned above as needing some adjustments?

**Comment by arcangelini** [comment]
I tested on Simple sites with paid plans and everything was working fine. The issue would be with Atomic sites. The public-api is not open to requests from AT sites since it is no longer an internal request (coming from WPCOM site). We have the ability to make requests using a Jetpack proxy but it requires the endpoint use v2 not v1.1.

I will work on fixing it this week. Is there a time that you were expecting this to be delivered by? @kriskarkoski

**Comment by alshakero** [comment]
> @alshakero Can you clarify what the issues are? We're sending add-on only users to the forums as of earlier this month and I'm not aware of issues.

Now that Tony has tested, we're good. It was nothing major, just that when we built the forum version of contact form, we assumed the user will always have a free site and never tested in a paid site and I thought there may be some issues with paid sites. But it seems to work in paid sites as well! We'll just need to handle atomic.

**Comment by kriskarkoski** [comment]
> I will work on fixing it this week. Is there a time that you were expecting this to be delivered by?

As quick as we can reasonably make happen, but no particular deadline. Jan. 31 was the deadline for the project getting everything moving, but we're relying on outside help like Vertex for some of the pieces and know this isn't the only thing on your plates cc @correliebre

**Comment by arcangelini** [comment]
Created a new v2 endpoint to handle our Atomic requests - D100588-code

**Review by alshakero — APPROVED** [review]
Tested in Simple and Atomic. Works great, thanks!

File: apps/editing-toolkit/editing-toolkit-plugin/help-center/class-help-center.php [hunk 1]
```diff
@@ -141,7 +140,7 @@ function ( $product ) {
 			'plan'            => array(
 				'product_slug' => $plan->product_slug,
 			),
-			'is_wpcom_atomic' => boolval( $at_options ),
+			'is_wpcom_atomic' => boolval( defined( 'IS_ATOMIC' ) && IS_ATOMIC ),
```
Comment (obenland) [inline apps/editing-toolkit/editing-toolkit-plugin/help-center/class-help-center.php]: With the switch to the conditional, `boolval()` becomes redundant

Comment (arcangelini) [reply apps/editing-toolkit/editing-toolkit-plugin/help-center/class-help-center.php]: Thanks!

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7817066

Thank you @arcangelini for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7817066).

**Pull request merged**